lib/pythontarget.c

```diff
@@ -679,6 +725,25 @@ action_capsule_init(generic_action_capsule_struct *self, PyObject *args, PyObjec
     return 0;
 }
 
+
+///////////////// Functions used in Tag creation, initialization and deletion /////////////
+/**
+ * Initialize the Tag object with the given values for "time" and "microstep", 
+ * both of which are required.
+ * @param self A py_tag_t object.
+ * @param args The arguments are:
+ *      @param time A logical time.
+ *      @param microstep A microstep within the logical time "time".
+ */
+static int Tag_init(py_tag_t *self, PyObject *args, PyObject *kwds)
+{
+    static char *kwlist[] = {"time", "microstep", NULL};
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "Lk", kwlist,
+                                     &(self->tag.time), &(self->tag.microstep)))
+        return -1;
```

Comment:
I think since the return statement is not on the same line as the `if` statement, there should be an accompanying `{ }` to avoid weird bugs such as [this](https://blog.codecentric.de/en/2014/02/curly-braces/)